test/integration/targets/acl/tasks/main.yml

```diff
@@ -33,4 +33,4 @@
     test_user: ansible_user
     test_group: ansible_group
     test_file: /tmp/ansible_file
```

Comment:
Should we change test_file to have spaces as well?

Reply:
No, because shlex.quote() behaves differently in each cases, it only escapes when needed.

```python
>>> shlex.quote('/tmp/file')
'/tmp/file'
>>> shlex.quote('/tmp/file space')
"'/tmp/file space'"
```